### ISSUE TITLE
Classification example

### ISSUE THREAD

**Issue by Scitator** [ieee8023/covid-chestxray-dataset#38]
Hi,

I have prepared an example of Xray classification , based on this repo
https://colab.research.google.com/drive/1KlKvHDgvi-cfrpJUIOCvczmL4Ctc1wBL

Please check it out. Are you interested in such initiatives?

**Comment by oplatek** [comment]
Hi,
we prepared DenseNet model using Pytorch Lightning

See https://github.com/ieee8023/covid-chestxray-dataset/issues/38

**Comment by Scitator** [comment]
@oplatek Several suggestions (just my thoughts during example preparation)

1. Metadata have a bunch of useful information; not only covid-19, but other disease labels also - you need to use them for better machine learning model
1. Due to class disbalance, `accuracy` is not significant metric - use `AUC` or `F1 score` instead
1. Due to low amount of data (only around 100 images) data augmentation is important to prevent overfitting
1. Because of medicine field, reproducibility really matters it this case; but this is [Catalyst](https://github.com/catalyst-team/catalyst) duty

long story short, all these DL best practices were implemented during example preparation,
https://colab.research.google.com/drive/1KlKvHDgvi-cfrpJUIOCvczmL4Ctc1wBL
Make the world a better with better science :)

**Comment by oplatek** [comment]
@Scitator thanks for your points!
Replacing accuracy is so obvious and it should be there even for the baseline model.
